File: JavaScript/867. Transpose Matrix.js
/**
 * @param {number[][]} matrix
 * @return {number[][]}
 */
var transpose = function(matrix) {
    var a = [];
    for(var i = 0;i < matrix[0].length;i++){
        var b = [];
        for(var j = 0;j < matrix.length;j++){
            b.push(matrix[j][i]);
        }
        a.push(b);
    }
    return a;
};

File: Python/867. Transpose Matrix.py
class Solution:
    def transpose(self, matrix: List[List[int]]) -> List[List[int]]:
        # hoge = []
        # for i in range(len(matrix[0])):
        #     fuga = []
        #     for j in range(len(matrix)):
        #         fuga.append(matrix[j][i])
        #     hoge.append(fuga)
        # return hoge
        
        return list(zip(*matrix))
